message_definitions/v1.0/common.xml

```diff
@@ -6195,6 +6195,12 @@
       <field type="uint32_t" name="active" display="bitmask">Active outputs</field>
       <field type="float[32]" name="actuator">Servo / motor output array values. Zero values indicate unused channels.</field>
     </message>
+    <message id="376" name="RELAY_STATUS">
+      <description>Reports the on/off state of relays, as controlled by MAV_CMD_DO_SET_RELAY.</description>
+      <field type="uint32_t" name="time_boot_ms" units="ms">Timestamp (time since system boot).</field>
+      <field type="uint16_t" name="present" display="bitmask">Relay present.  Relay instance numbers are represented as individual bits in this mask by offset.  Bits will be true if a relay instance is configured.</field>
```

Comment:
I'm a little skeptical on the value here. I'm not opposed, but it seems like for a user to meaningfully interpret this information they need out of band data anyways to know what a given relay will do. It does let a GCS remove showing a relay from it's UI though, which I can see the value for.

Can we flip these two fields, so that 0 stripping will generally get to remove the last byte of `present`? It's undefined what the relay is reported as if it's not present, but not present being 0 means that anyone using less then 8 relays saves a byte?

Reply:
Flipped the fields.